jupyter-js-widgets/src/manager-base.ts

```diff
@@ -424,11 +425,13 @@ abstract class ManagerBase<T> {
 
                 // First put back the binary buffers
                 let decode = {'base64': toByteArray, 'hex': utils.hexToBuffer};
-                let modelState = models[model_id].state;
-                if (modelState.buffers) {
-                    let bufferPaths = modelState.buffers.map(b => b.path);
-                    let buffers = modelState.buffers.map(b => decode[b.encoding](b.data));
-                    utils.put_buffers(modelState.state, bufferPaths, buffers);
+                let model = models[model_id];
+                let modelState = model.state;
+                if (model.buffers) {
+                    let bufferPaths = model.buffers.map(b => b.path);
+                    // similas as what comes off the wire, we put the data into a DataView object
+                    let buffers = model.buffers.map(b => new DataView(decode[b.encoding](b.data).buffer));
```

Comment:
Another great catch! I didn't remember that the result of the binary deserialization was that buffers were DataViews.